components/Posts.php

```diff
@@ -175,13 +182,14 @@ protected function listPosts()
         $isPublished = !$this->checkEditor();
 
         $posts = BlogPost::with('categories')->listFrontEnd([
-            'page'       => $this->property('pageNumber'),
-            'sort'       => $this->property('sortOrder'),
-            'perPage'    => $this->property('postsPerPage'),
-            'search'     => trim(input('search')),
-            'category'   => $category,
-            'published'  => $isPublished,
-            'exceptPost' => $this->property('exceptPost'),
+            'page'             => $this->property('pageNumber'),
+            'sort'             => $this->property('sortOrder'),
+            'perPage'          => $this->property('postsPerPage'),
+            'search'           => trim(input('search')),
+            'category'         => $category,
+            'published'        => $isPublished,
+            'exceptPost'       => $this->property('exceptPost'),
+            'exceptCategories' => explode(',', $this->property('exceptCategories')),
```

Comment:
Try this to accept array format:
```
'exceptCategories' => is_array($this->property('exceptCategories'))
                ? $this->property('exceptCategories')
                : explode(',', $this->property('exceptCategories')),
```

And in view, use like this:

`exceptCategories[] = "ebooks"`

Reply:
I've integrated that change since the syntax
```
exceptCategories[] = "ebooks"
exceptCategories[] = "music"
```
is much nicer than:
```
exceptCategories = "ebooks,music"
```
@silasrm Thanks for the suggestion.